src/extension/src/manifest.xml

```diff
@@ -2,7 +2,7 @@
 <ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
   <ProviderNameSpace>Microsoft.CPlat.Core</ProviderNameSpace>
   <Type>LinuxPatchExtension</Type>
-  <Version>1.6.41</Version>
```

Comment:
is this the only file need to be updated for new version? In WPE, we also update the ExtensionInfo.xml file. this is the file that will be used when we publish the extension